coral-dbt/src/main/resources/macros/coral_macros/default/utils/get_coral_incremental_response.sql

```diff
@@ -3,11 +3,21 @@
  * Licensed under the BSD-2 Clause license.
  * See LICENSE in the project root for license information.
  */
-{% macro get_coral_incremental_response(sql, table_names) %}
-  {{ return(adapter.dispatch('get_coral_incremental_response', macro_namespace = 'coral_dbt')(sql, table_names)) }}
+{% macro get_coral_incremental_response(sql, url, table_names, sql_dialect) %}
```

Comment:
as part of the macro documentation, lets also capture the request & response data with an example here.